### DISCUSSION TITLE
deck.gl data collection

### DISCUSSION THREAD

**Discussion by obones** [visgl/deck.gl#5329] [Q&A] [answered]
Hello,

I'm currently evaluation deck.gl for a map based visualization project and I am wondering if there is any data collection made while using. Something like usage metrics, behavior statistics and the like.
I understand that the map tiles are provided by a third party (MapTiler, MapBox, Google) so I'll have to gather the same information from them, but I'm interested in deck.gl own behavior.
From what I could figure out from the documentation and the code, there is no such data collection made by deck.gl.

Is my conclusion correct?

Thanks for your answers

**Comment by Pessimistress** [comment, marked as answer]
deck.gl is an Urban Computing Foundation project. It is not a commercial product, and does not use any service to track usage.

You can collect metrics for your own app. See [metrics](https://deck.gl/docs/api-reference/core/deck#metrics).

**Reply by obones** [reply]
Thanks, that's perfect
